Ecommerce/index.css

```diff
@@ -1 +1,5 @@
-/* Write code below for ECOMMERCE TASK */
+/* Write code below for ECOMMERCE TASK */
+img {

```

Comment:
The 'justify-content' property is used for flex containers, not images. This line should be removed.